### ISSUE TITLE
Remove confusing old git tags in the repository 

### ISSUE THREAD

**Issue by Skunnyk** [scylladb/gocql#110]
Hi,

In the git repository (https://github.com/scylladb/gocql/tags), there are old git tags with higher digit number than current release ! 
Some dependency tools (like renovate bot) are confused and get release `1.10.0` for example.
Last scylladb/gocql official release is 1.7.3, but we have tags numbered 1.8.0 or 1.10.0 from 2021! This is confusing.

Is it possible to remove theses old tags or do something ? :) 

Thank you !

**Comment by martin-sucha** [comment]
Beware that removing tags will not remove the tags from git clones nor cached packages from module proxies. It might be safer to just release a new version that has higher version than all previous tags. And mark the unwanted versions as retracted in the Go module.

**Comment by Skunnyk** [comment]
Yes, deleting old tags can be a problem. So next version slot is 1.11.0 then… :)

**Comment by carlosms** [comment]
Until this is fixed, please consider updating the instructions in the readme:
https://github.com/scylladb/gocql/blob/e38b2bcca0a61280272806d1e2bd2b23c9749773/README.md?plain=1#L23-L35

This will put `v1.10.0` in the go.mod file.

In my opinion it might be simpler for everyone to release a higher version number right now, even if it's the exact same commit as 1.7.3.
